src-python/lab-service/traffexam/kilda/traffexam/model.py

```diff
@@ -360,6 +360,19 @@ def __init__(self, mac_address, port_number, chassis_id, ttl, **fields):
         self.time_to_live = ttl
 
 
+class LACPPush(Abstract):
+    def __init__(self, payload):
+        super().__init__(**{})
+        self.expired = payload["expired"]
```

Comment:
Why did you change the init principle?

Reply:
Please, check if the new version matches the common applicaton approach.